Library/Homebrew/sbom.rb

```diff
@@ -0,0 +1,348 @@
+# typed: true
+# frozen_string_literal: true
+
+require "json_schemer"
```

Comment:
@SMillerDev This will only be available for `dev-cmd` so may need to get moved into a method call instead. I suspect that's needed to fix CI failures.

Reply:
I'm not sure what you mean by a method call, do you mean to make a command to generate an SBOM?

Reply:
Move the `require` inside a `def something`